operations/helm/charts/mimir-distributed/templates/metamonitoring/grafana-agent.yaml

```diff
@@ -0,0 +1,97 @@
+{{- with .Values.metaMonitoring.grafanaAgent }}
+{{- if .enabled }}
+apiVersion: monitoring.grafana.com/v1alpha1
+kind: GrafanaAgent
+metadata:
+  name: {{ include "mimir.resourceName" (dict "ctx" $ "component" "meta-monitoring") }}
+  namespace: {{ .namespace | default $.Release.Namespace | quote }}
+  labels:
+    {{- include "mimir.labels" (dict "ctx" $ "component" "meta-monitoring" ) | nindent 4 }}
+    {{- with .labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  {{- with .annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  serviceAccountName: {{ include "mimir.resourceName" (dict "ctx" $ "component" "grafana-agent") }}
+  logs:
+    instanceSelector:
+      matchLabels:
+        {{- include "mimir.selectorLabels" (dict "ctx" $ "component" "meta-monitoring") | nindent 8 }}
+    # cluster label for logs is added in the LogsInstance
+  metrics:
+    instanceSelector:
+      matchLabels:
+        {{- include "mimir.selectorLabels" (dict "ctx" $ "component" "meta-monitoring") | nindent 8 }}
+    externalLabels:
+      cluster: {{ include "mimir.clusterName" $ }}
+
+---
```

Comment:
Would be nice to split these into files for clarity and alignment with the rest of the chart.

Reply:
done in 15c3706103563ffe02317f79d11212201eaa8377